custom/icds/view_utils.py

```diff
@@ -32,7 +32,7 @@ def is_icds_cas_project(domain):
 def check_authorization(domain, user, app):
     if (
         settings.SERVER_ENVIRONMENT == 'icds-staging'
-        and toggles.ROLE_WEBAPPS_PERMISSIONS.enabled(domain)
+        and toggles.ROLE_WEBAPPS_PERMISSIONS.enabled(user.username)
```

Comment:
if you aren't going to check the domain here you should drop the domain namespace in the toggle definition

Reply:
There is one other place where toggle is used via domain namespace and I think we will remove user namespace once we confirm everything is working as intended.